README.md

```diff
@@ -22,124 +22,132 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
+| 0018 | [Multi-Stake-Keys Wallets](./CIP-0018/) | Draft |
+| 0019 | [Cardano Addresses](./CIP-0019/) | Active |
+| 0020 | [Transaction message/comment metadata](./CIP-0020/) | Active |
+| 0021 | [Transaction requirements for interoperability with hardware wallets](./CIP-0021/) | Draft |
+| 0022 | [Pool operator verification](./CIP-0022/) | Active |
+| 0023 | [Fair Min Fees](./CIP-0023/) | Draft |
+| 0024 | [Non-Centralizing Rankings](./CIP-0024/) | Draft |
+| 0025 | [Media NFT Metadata Standard](./CIP-0025/) | Active |
+| 0026 | [Cardano Off-Chain Metadata](./CIP-0026/) | Draft |
+| 0027 | [CNFT Community Royalties Standard](./CIP-0027/) | Draft |
+| 0028 | [Protocol Parameters (Alonzo Era)](./CIP-0028/) | Active |
+| 0029 | [Phase-1 Monetary Scripts Serialization Formats](./CIP-0029/) | Active |
+| 0030 | [Cardano dApp-Wallet Web Bridge](./CIP-0030/) | Active |
+| 0031 | [Reference Inputs](./CIP-0031/) | Active |
+| 0032 | [Inline Datums](./CIP-0032/) | Active |
+| 0033 | [Reference Scripts](./CIP-0033/) | Active |
+| 0034 | [Chain ID Registry](./CIP-0034/) | Draft |
+| 0035 | [Plutus Core Evolution](./CIP-0035) | Active |
+| 0036 | [Catalyst/Voltaire Registration Transaction Metadata Format](./CIP-0036) | Proposed |
+| 0037 | [Dynamic Saturation Based on Pledge](./CIP-0037) | Proposed |
+| 0040 | [Collateral Output](./CIP-0040) | Active |
+| 0042 | [New Plutus Builtin: serialiseBuiltinData](./CIP-0042) | Active |
+| 0049 | [ECDSA and Schnorr signatures in Plutus Core](./CIP-0049) | Proposed |
+| 0052 | [Cardano Audit Best Practice Guidelines](./CIP-0052) | Proposed |
+| 0054 | [Cardano Smart NFTs](./CIP-0054) | Draft |
+| 0055 | [Protocol Parameters (Babbage Era)](./CIP-0055) | Active |
+| 0057 | [Plutus Smart-Contract Blueprint](./CIP-0057) | Active |
+| 0058 | [Plutus Bitwise Primitives](./CIP-0058) | Proposed |
+| 0059 | [Terminology Surrounding Core Features](./CIP-0059) | Active |
+| 0060 | [Music Token Metadata](./CIP-0060) | Active |
+| 0067 | [Asset Name Label Registry](./CIP-0067) | Proposed |
+| 0068 | [Datum Metadata Standard](./CIP-0068) | Proposed |
+| 0069 | [Plutus Script Type Uniformization](./CIP-0069) | Proposed |
+| 0071 | [Non-Fungible Token (NFT) Proxy Voting Standard](./CIP-0071) | Proposed |
+| 0072 | [DApp Registration](./CIP-0072) | Proposed |
+| 0074 | [Set min-pool-cost to 0](./CIP-0074) | Proposed |
+| 0075 | <strike>[Fair Stake Pool Rewards](./CIP-0075)</strike> | Inactive |
+| 0080 | [Transaction Serialization Deprecation Cycle](./CIP-0080) | Active |
+| 0082 | <strike>[Improved Rewards Scheme Parameters](./CIP-0082)</strike> | Inactive |
+| 0083 | [Encrypted Transaction message/comment metadata (Addendum to CIP-0020)](./CIP-0083) | Active |
+| 0085 | [Sums-of-products in Plutus Core](./CIP-0085) | Proposed |
+| 0086 | [NFT Metadata Update Oracles](./CIP-0086) | Proposed |
+| 0094 | [SPO On-chain Polls](./CIP-0094) | Active |
+| 0381 | [Plutus Support for Pairings Over BLS12-381](./CIP-0381) | Proposed |
```

Comment:
Sorry for this. This change is probably unnecessary though it had bothered me for a long time. I can revert back to numbers with no-leading zero if someone has strong feeling about it. 

The diff is now a bit hard to read but the main diff are:

- Addition of CIP-0094 as Active 
- Addition of CIP-0069 as Proposed

Reply:
I think this change improves both readability and parse/sort-ability 👍